src/main/java/mate/academy/rickandmorty/controller/CharactersController.java

```diff
@@ -0,0 +1,34 @@
+package mate.academy.rickandmorty.controller;
+
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.dto.internal.CharacterInternalDto;
+import mate.academy.rickandmorty.service.character.CharacterService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/characters")
+@RequiredArgsConstructor
+@Tag(name = "Characters management",
+        description = "Endpoints for managing characters from the Rick and Morty cartoon")
```

Comment:
Usually `managing` means that the API allows some creation/modification, but this API is only for read access.
```suggestion
        description = "Endpoints for retrieving characters from the Rick and Morty cartoon")
```